### DISCUSSION TITLE
Missing quarter in address information (JSON queries)

### DISCUSSION THREAD

**Discussion by mod3330** [osm-search/Nominatim#3547] [Q&A]
In the responses to the query, "Otemachi" (https://www.openstreetmap.org/relation/3545196) appears as part of the `display_name` but not under `address`:

https://nominatim.openstreetmap.org/reverse?format=json&lat=35.68965&lon=139.7612&zoom=18

It does appear as part of the address in the web UI:

https://nominatim.openstreetmap.org/ui/details.html?osmtype=N&osmid=6438895388&class=man_made

The same problem occurs when using XML or GeocodeJSON. This apparently has to do with the boundary being tagged `place=quarter`. For example, in another query to a nearby coordinate the respective boundary at the same admin level (外神田 - Soto-Kanda, missing the `place=quarter` tag) appears in the address details as `suburb`:

https://nominatim.openstreetmap.org/reverse?format=json&lat=35.7008&lon=139.77219&zoom=18

**Comment by lonvia** [comment]
https://www.openstreetmap.org/relation/3545196 is classified as 'neighbourhood' in the end because it has the same wikidata tag as https://www.openstreetmap.org/node/8354653212. Clearly one of the two objects has a  wrong wikidata tag. The wikidata tags on the chome nodes possibly need deleting.

So, long story short, this is a data issue. If you are interested in admin boundaries by level, use `format=geocodejson`.
